xsuite-chainsimulator/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@xsuite/chainsimulator",
+  "version": "0.0.1",
+  "license": "MIT",
+  "scripts": {
+    "build": "run-script-os",
+    "build:linux": "BINARY=csproxy-linux-amd64 ./build-chainsimulator.sh",
```

Comment:
Not good and same for line below

Reply:
I don't understand here, the binary compiles OK like this for Linux, it did not work with the other ldflags.
This works fine.

Reply:
Done